outrank/algorithms/importance_estimator.py

```diff
@@ -38,12 +38,14 @@ def sklearn_MI(vector_first: Any, vector_second: Any) -> float:
 
 
 def sklearn_surrogate(
-    vector_first: Any, vector_second: Any, surrogate_model: str,
+    vector_first: Any, vector_second: Any, X: Any, surrogate_model: str
 ) -> float:
-    if surrogate_model == 'surrogate-LR':
```

Comment:
These if-elses could be abstracted away into a modelfactory that returns initialized class w.r.t. str. input